modules/securityhub/main.tf

```diff
@@ -62,3 +62,160 @@ resource "aws_securityhub_standards_control" "pci_disable_ensure_mfa_for_root" {
   disabled_reason       = "Root login actions prevented with SCPs"
   depends_on            = [aws_securityhub_standards_subscription.pci]
 }
+
+# SecurityHub Alerting
+
+# Filter for New, High & Critical SecHub findings but exclude Inspector
+resource "aws_cloudwatch_event_rule" "sechub-high-and-critical-findings" {
```

Comment:
Can I please ask we rename the resource to use underscores instead of hyphens in line with the style of the rest of this file?

Reply:
Thanks David - good shout